### PR TITLE
TASK: Quick Start in Readme.rst

### PR DESCRIPTION
**Pull request by christoph-daehne**
The Readme.rst contains a section explaining how to clone the project and create patches. The explanation is very brief and links to more detailed guides.

### CODE REVIEW

File: Readme.rst [hunk 1]
```diff
@@ -7,3 +7,36 @@ the development collections of Neos and Flow, which allows you to create bugfixe
 the respective repositories for review and inclusion into the core.
 
 Learn more about the Neos content application platform on http://www.neos.io/.
+
+Quick Start
+===========
+
+local checkout
```
Comment (kdambekalns) [inline Readme.rst]: Local Checkout

File: Readme.rst [hunk 2]
```diff
@@ -7,3 +7,36 @@ the development collections of Neos and Flow, which allows you to create bugfixe
...
+
+  cd /your/local/path
+  git clone https://github.com/neos/neos-development-distribution.git
+  cd neos-development-distribution
+  curl -s https://getcomposer.org/installer | php
+  php composer.phar install
+
+For details see https://discuss.neos.io/t/development-setup
+
+web server configuration
```
Comment (kdambekalns) [inline Readme.rst]: Web Server Configuration

File: Readme.rst [hunk 3]
```diff
@@ -7,3 +7,36 @@ the development collections of Neos and Flow, which allows you to create bugfixe
...
+------------------------
+
+See https://www.neos.io/develop/download.html#setting-up-neos
+
+how to push changes
```
Comment (kdambekalns) [inline Readme.rst]: Ho to Push Changes

File: Readme.rst (hunk 1 above)
Comment (albe) [reply Readme.rst:14]:
```suggestion
Local Checkout
```

File: Readme.rst (hunk 2 above)
Comment (albe) [reply Readme.rst:27]:
```suggestion
Web Server Configuration
```

File: Readme.rst (hunk 3 above)
Comment (albe) [reply Readme.rst:32]:
```suggestion
How to Push Changes
```

**Comment by albe** [comment]
Meh, I thought suggestions would allow to change the code directly... :/ Suggestion: Let's merge this as is and add a new PR with the fixes

**Comment by kdambekalns** [comment]
> Meh, I thought suggestions would allow to change the code directly... 

They do - but you need to be the assignee to be able to accept them :)

**Comment by albe** [comment]
Just being assigned is not enough apparently. Seems back then only committer could do that. Oh well. I'll merge now and take care of the fix-PR then.

**Pull request merged**